velox/expression/fuzzer/DecimalArgGeneratorBase.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/expression/fuzzer/ArgGenerator.h"
+
+namespace facebook::velox::fuzzer {
+
+/// An argument type generator for decimal function. A map keyed on the pair of
```

Comment:
Since this is a base class, perhaps, explain how to define derived classes.

Reply:
Added below sentence in the comment.
> Derived classes should supply implementations for those methods that need to be overriden.

Reply:
> Derived classes should supply implementations for those methods that need to be overriden.

This is a bit cryptic and requires the developer to read and understand how this class works. Can we provide a bit more help by saying something like:

```
Derived classes should call 'initialize' from the constructor and specify the number of decimal arguments. They should also implement toReturnType with matching number of pairs of precision and scale.
```

Reply:
I see. That's is much better. Thanks!